PowerEditor/src/MISC/PluginsManager/Notepad_plus_msgs.h

```diff
@@ -36,8 +36,10 @@ enum LangType {L_TEXT, L_PHP , L_C, L_CPP, L_CS, L_OBJC, L_JAVA, L_RC,\
 			   // Don't use L_JS, use L_JAVASCRIPT instead
 			   // The end of enumated language type, so it should be always at the end
 			   L_EXTERNAL};
+enum class LangAutoIndentMode { Standard, Extended, Custom };

```

Comment:
Because _Notepad_plus_msgs.h_ is an interface file,  so I think it would be better to make this enum more explicit, it helps not only the plugin authors, it's useful also for maintaining the code. As a result, it would be better for me to rename it to:
`enum class ExternalLexerAutoIndentMode { Basic, C_Like, Custom };`

Sorry for changing the naming again.

Reply:
Again, not a problem. Better if its purpose stay as clear as possible. :)